ui/component/or-map/src/index.ts

```diff
@@ -518,6 +519,14 @@ export class OrMap extends LitElement {
                     this._processNewMarkers(info.addedNodes);
                     this._processRemovedMarkers(info.removedNodes);
                 });
+                this._resizeObserver?.disconnect();
+                this._resizeObserver = new ResizeObserver(debounce(() => {
+                    this.resize();
+                }, 20));
```

Comment:
20ms might be quite fast / performance heavy. 200ms is more suitable. You can do some testing how many times it executes the function with logging. I at least had some experiences where values under 100ms would just trigger infinitely whilst it was unnecessairy.